pages/querying/schema.mdx

```diff
@@ -7,11 +7,252 @@ import { Callout } from 'nextra/components'
 
 # Schema
 
-Some queries are designed to retrieve information about the structure of the data within the database, for example, node labels and properties associated, as well as relationship types and properties.
+Schema queries are designed to retrieve information about the structure of the data within the database, for example, node labels and properties associated, as well as relationship types and properties.
 
-## Schema-related queries
+## Run-time schema tracking
 
-To use schema-associated queries, start Memgraph with the `--storage-enable-schema-metadata` [configuration flag](/configuration/configuration-settings#storage) set to `True`. The flag facilitates the utilization of a specialized cache designed to store specific metadata related to the database.   
+To use run-time schema tracking, start Memgraph with the `--schema-info-enabled` [configuration flag](/configuration/configuration-settings#storage) set to `True`. Once enabled, every change to the data layout will be tracked, and a derived schema will be automatically updated. Note that enabling this feature may result in a slight performance decrease; for more details, refer to [Performance penalty](#performance-penalty).
+
+### What Schema Information is Tracked?
+
+The runtime schema tracks all uniquely defined nodes and edges:
+
+* __Nodes__ are defined by their labels.
+* __Edges__ are defined by their type, and the labels of their source and destination nodes.
+
+For each unique object (node or edge), the schema tracks its count and categorizes stored properties by key and type.
+
+### Querying the Schema
+
+You can query the current schema by running  `SHOW SCHEMA INFO`. This returns a JSON representation of the schema, divided into six sections:
+
+* `nodes` - contains a set of uniquely identified nodes
+* `edges` - contains a set of uniquely identified edges
+* `node_constraints` - list of all currently defined node constraints
+* `node_indexes` - list of all currently defined node indexes
+* `edge_indexes` - list of all currently defined edge indexes
+* `enums` - list of all currently defined enums and their values
+
+#### Exceptions
+
+Exceptions that might be thrown while executing `SHOW SCHEMA INFO` query:
+* `Show schema info is not supported for OnDisk.` - Please use InMemory Transactional or Analytical mode instead.
+* `Show schema info cannot be executed in multicommand transactions.` - Show schema must be executed in an [implicit (or auto-commit) transaction](#implicit-transactions).
```

Comment:
I can't find this #implicit-transactions anchor either

Reply:
removed the link